docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -0,0 +1,585 @@
+:linkTroubleshooting: https://github.com/integr8ly/example-customisations/blob/master/docs/troubleshooting.adoc
+:linkGitHubFork: https://help.github.com/articles/fork-a-repo/
+:linkIndexURL: https://github.com/integr8ly/example-customisations/index.adoc
+:walkthrough-1A: *Integrating message-oriented middleware with a RESTful API using AMQ Online*
+:walkthrough-2: *Managing and protecting APIs using API management*
+:publishing-walkthrough: *Adding walkthroughs to a cluster*
+
+= Writing {PRODUCT} Walkthroughs
+
+
+== About Walkthroughs
+
+
+Walkthroughs are tutorials available in {PRODUCT}. The default walkthroughs guide you through the following scenarios:
+
+* {walkthrough-1A}
+* {walkthrough-2}
+
+These walkthroughs are available in the {web-app}.
+
+In addition to the default walkthroughs, you can import existing walkthroughs from a Git repository. 
```

Comment:
I would say something like " To add walkthroughs to your cluster, see <walkthrough name>." The fact that it is is a walkthrough in itself is information that is not needed at this time.

Reply:
hmmm, it kinda is the point here, but I rewrote section to move it into list of walkthroughs